examples/ota-requestor-app/linux/main.cpp

```diff
@@ -89,7 +92,10 @@ OptionSet cmdLineOptions = { HandleOptions, cmdLineOptionsDef, "PROGRAM OPTIONS"
                              "        advertisements. If none is specified, default value is 3840.\n"
                              "  -q/--delayQuery <Time in seconds>\n"
                              "        From boot up, the amount of time to wait before triggering the QueryImage\n"
-                             "        command. If none or zero is supplied, QueryImage will not be triggered.\n" };
+                             "        command. If none or zero is supplied, QueryImage will not be triggered.\n"
+                             "  -c/--RequestorCanConsent\n"
+                             "        If provided, the RequestorCanConsent field of the Query Image request is set to true.\n"
+                             "        Else, it is set to false.\n " };
```

Comment:
Nitpck: If this arg is not provided, wouldn't the QueryImage-field be set to whatever value is implemented/returned by CanConsent()?